src/site/content/en/blog/isinputpending/index.md

```diff
@@ -0,0 +1,194 @@
+---
+title: Better JS scheduling with isInputPending()
+subhead: >
+  A new JavaScript API that may help you avoid the tradeoff between
+  load performance and input responsiveness.
+description: >
+  A new JavaScript API that may help you avoid the tradeoff between
+  load performance and input responsiveness.
+date: 2020-11-13
+hero: hero.jpg
+alt: A traffic sign that says 'GIVE WAY'.
+tags:
+  - blog
+  - javascript
+  - performance
+  - rendering
+authors:
+  - nateschloss
+  - andrewcomminos
+---
+
+Loading fast is hard. Sites that leverage JS to render their content currently
+have to make a trade-off- either perform all the work needed for display all at
+once, or chunk the work into smaller tasks in order to remain responsive to
+input and paint -- occasionally, at the expense of performance.
+
+To eliminate the need to make this trade-off, Facebook proposed and implemented
+the `isInputPending()` API in Chrome in order to improve responsiveness without
+yielding. Based on origin trial feedback, we've made a number of updates to the
+API, and are happy to announce that the API is now shipping by default in Chrome
+87!
+
+## Background
```

Comment:
```suggestion
87!

## Browser compatibility

`isInputPending()` is shipping in Chromium-based browsers starting in version 87.

## Background
```

We know that browser compatibility is a big concern for developers (https://web.dev/handbook/content-checklist/#compatibility). We should add a browser compat section to the start of the page as a courtesy. What's the status in other browsers? We usually either say "no signal" or "browser X is interested in implementing" or "browser X is implementing" or "browser X has shipped"